src/Microsoft.ML.Transforms/Text/StopWordsRemoverTransform.cs

```diff
@@ -229,8 +237,8 @@ private static VersionInfo GetVersionInfo()
         {
             return new VersionInfo(
                 modelSignature: "STOPWRDR",
-                verWrittenCur: 0x00010001, // Initial
-                verReadableCur: 0x00010001,
+                verWrittenCur: 0x00010002, // Initial
```

Comment:
Instead of just

```csharp
verWrittenCur: 0x00010002, // Initial
```

I would prefer something more along these lines.

```csharp
//verWrittenCur: 0x00010002, // Initial
verWrittenCur: 0x00010001, // Corrected Norwegian Bokmål stopwords.
```

The idea is that as more versions are added, you have a little running catalog of why each version bump was necessary. The most extreme example of this is in our `TextLoader`, that is:

[https://github.com/dotnet/machinelearning/blob/fc7286c7d9aa9218c4c8da3357b1c58361c7e8f5/src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs#L909-L919](https://github.com/dotnet/machinelearning/blob/fc7286c7d9aa9218c4c8da3357b1c58361c7e8f5/src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs#L909-L919)